source/_integrations/bayesian.markdown

```diff
@@ -199,3 +207,66 @@ binary_sensor:
 ```
 
 {% endraw %}
+
+### Multiple state and numeric entries per entity
+
+Lastly an example that illustrates how to configure Bayesian when there are more than two states of interest and a number of possible numeric ranges. When an entity can hold more than 2 values of interest (numeric ranges or states) then you may wish to specify probabilities for each possible value. Once you have specified more than one, Bayesian cannot infer anything about states or numeric values that are unspecified, like it usually does, so it is recommended that all possible values are included. As above the `prob_given_true`s of all the possible states should sum to 1, as should the `prob_given_false`s. If a value is observed that has not been specified then the observation will be ignored as it would if the entity were `UNKNOWN` or `UNAVAILABLE`.
```

Comment:
```suggestion
Lastly, an example illustrates how to configure Bayesian when there are more than two states of interest and several possible numeric ranges. When an entity can hold more than 2 values of interest (numeric ranges or states), then you may wish to specify probabilities for each possible value. Once you have specified more than one, Bayesian cannot infer anything about states or numeric values that are unspecified, like it usually does, so it is recommended that all possible values are included. As above, the `prob_given_true`s of all the possible states should sum to 1, as should the `prob_given_false`s. If a value that has not been specified is observed, then the observation will be ignored as it would be if the entity were `UNKNOWN` or `UNAVAILABLE`.
```